crypto/ocsp/ocsp_test.cc

```diff
@@ -1115,3 +1121,91 @@ TEST_P(OCSPURLTest, OCSPParseURL) {
     EXPECT_FALSE(path);
   }
 }
+
+// Nonce value used in "crypto/ocsp/test/aws/ocsp_response.der".
+static unsigned char ocsp_response_nonce[] = {
+    0xaf, 0xab, 0xd4, 0xec, 0x6a, 0x17, 0x2c, 0x4a,
+    0x98, 0xfb, 0x1a, 0x6d, 0x22, 0xff, 0x29, 0x28};
+
+struct OCSPNonceTestVector {
+  const char *ocsp_request;
+  const char *ocsp_response;
+  bool add_nonce;
+  unsigned char *nonce;
+  int nonce_len;
+  int nonce_check_status;
+};
+
+static const OCSPNonceTestVector kNonceTestVectors[] = {
+    // Add a nonce to an OCSP request without a nonce.
+    {"ocsp_request_no_nonce", "ocsp_response", true, ocsp_response_nonce,
+     sizeof(ocsp_response_nonce), OCSP_NONCE_EQUAL},
+    // Add a nonce to an OCSP request with a nonce. The original nonce should be
+    // overwritten.
+    {"ocsp_request", "ocsp_response", true, ocsp_response_nonce,
+     sizeof(ocsp_response_nonce), OCSP_NONCE_EQUAL},
+    // Generate a random nonce, which should be different from the nonce
+    // available in the OCSP response (unless we have a collision).
+    {"ocsp_request", "ocsp_response", true, nullptr, 0, OCSP_NONCE_NOT_EQUAL},
+    // OCSP request with no nonce, but an OCSP response that does have one.
+    {"ocsp_request_no_nonce", "ocsp_response", false, nullptr, 0,
+     OCSP_NONCE_RESPONSE_ONLY},
+    // OCSP request with no nonce, but an OCSP response that does have one.
+    {"ocsp_request", "ocsp_response_no_nonce", false, nullptr, 0,
+     OCSP_NONCE_REQUEST_ONLY},
+    // OCSP request with no nonce, but an OCSP response that does have one.
+    {"ocsp_request_no_nonce", "ocsp_response_no_nonce", false, nullptr, 0,
+     OCSP_NONCE_BOTH_ABSENT},
+};
```

Comment:
comment don't seem to match the test cases

Reply:
Got caught with the copy and paste, and forgot to update